README.md

```diff
@@ -124,29 +124,43 @@ the FHIR store, its full URL would be:
 ## Compile and run the streaming app
 From the root of your git repo, run:
 
-`$ mvn clean install`
+`mvn clean -B install -DskipTests -pl streaming-atomfeed -am`
 
 and then:
 
 ```
-$ mvn exec:java -pl streaming-atomfeed \
+mvn exec:java -pl streaming-atomfeed \
     -Dexec.args=" --openmrsUserName=admin --openmrsPassword=Admin123 \
     --openmrsServerUrl=http://localhost:8099 \
-    --fhirSinkPath=projects/PROJECT/locations/LOCATION/datasets/DATASET/fhirStores/FHIRSTORENAME \
+    --fhirSinkPath=localhost:8098/fhir \
```

Comment:
Can we separate out the HAPI FHIR sink related changes from the atom-feed ones into two PRs?